### ISSUE TITLE
Different (outdated?) version on npm

### ISSUE THREAD

**Issue by olessavluk** [relayjs/relay-devtools#11]
Hello

When I open "Updates" I see the following error in the console: 
![DevTools Console](https://i.imgur.com/3L8eRJa.png)

However, when I built the latest snapshot from the master - it worked just fine. And this line of code is also different - [UpdatesView.js#L101](https://github.com/relayjs/relay-devtools/blob/master/src/frontend/components/UpdatesView.js#L101)

`relay-devtools` version in yarn.lock:
```
relay-devtools@^1.4.0:
  resolved "https://registry.yarnpkg.com/relay-devtools/-/relay-devtools-1.4.0.tgz#7456c4ed3ad1e06f559fd80a77fee17f9e8edf80"
```

So it looks like the current npm version is different or maybe outdated?

**Comment by kassens** [comment]
Yeah, I'll have to figure out how to update the version in the chrome store

**Comment by sibelius** [comment]
any update on this?

**Comment by modosc** [comment]
yeah, this is still an issue - we get `Uncaught Error: unknown call: "relayDebugger:getEnvironmentsDetails"` every time we startup. this method exists in github but not on what's shipped via npm.

also, any chance on getting instructions for doing a npm build in the readme?

**Comment by olessavluk** [comment]
@modosc you can build & run by doing:
```
yarn build:standalone
node lib/standalone/launchApp.js
```
